### PR TITLE
Add Wikipedia and Wikimedia Commons link to iOS Place Page

### PR DESCRIPTION
**Pull request by dvdmrtnz**
This PR implements https://github.com/organicmaps/organicmaps/issues/3755, adding Wikipedia and Wikimedia Commons link to iOS Place Page. Also I modified the Wikimedia Commons icon on Android to be blue like the rest of the icons.

| Android | iOS |
|-|-|
|  |  |

### CODE REVIEW

**Review by vng — APPROVED** [review]
LGTM.
Is it necessary for Android to keep drawable xml and resources-svg for the same time?

**Comment by dvdmrtnz** [comment]
> LGTM. UPD: Didn't noticed that svg is not in android, but separate data folder. Update data/resources-svg/README.md with wiki-commons icon source.

Done, added image source

File: data/resources-svg/README.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ Twitter icon is downloaded from https://about.twitter.com/en/who-we-are/brand-to
 VK icon is downloaded from https://vk.com/brand
 
 Line logo is downloaded from https://line.me/en/logo
+
+Wikimedia Commons logo is downloaded from https://commons.wikimedia.org/wiki/File:Commons-logo.svg
```
Comment (biodranik) [inline data/resources-svg/README.md:13]: Shouldn't it also be in data/copyright.html?

**Review by biodranik — COMMENTED** [review]
1. Did you test it on real data? Do we get full valid URLs in the UI for openUrl?
2. What do you think about reusing the existing Wikipedia page description and adding:
   1. A  "read on Wikipedia" link in the right corner, opposite to the "more..." button?
   2. A "read on Wikipedia" link at the end of the text which is open when you press "more..."?
  It may save one line of the vertical space in the Place Page for other useful info.

**Comment by dvdmrtnz** [comment]
> 1. Did you test it on real data? Do we get full valid URLs in the UI for openUrl?

I tested it using real data. It works. We do get full valid URLs for the openUrl.

>  2. What do you think about reusing the existing Wikipedia page description and adding:
>      1. A  "read on Wikipedia" link in the right corner, opposite to the "more..." button?
>      2. A "read on Wikipedia" link at the end of the text which is open when you press "more..."?
>          It may save one line of the vertical space in the Place Page for other useful info.

This seems interesting. However, I see one problem. Currently the Wikipedia page description is only shown in a handful of languages. Keeping the Wikipedia link separately allows us to show the link even when we do not show a Wikipedia description. For example, this POI has only a Basque language Wikipedia link and we do not show the Wikipedia page description: https://www.openstreetmap.org/node/807344081

**Comment by dvdmrtnz** [comment]
I just realized that on the iOS app we do not usually use full capital letters labels. Should I change "WIKIPEDIA" to "Wikipedia" and "WIKIMEDIA COMMONS" to "Wikimedia Commons"?

**Comment by biodranik** [comment]
> Keeping the Wikipedia link separately allows us to show the link even when we do not show a Wikipedia description.

Good point! Ok, let's leave it for the future.

> I just realized that on the iOS app we do not usually use full capital letters labels. Should I change "WIKIPEDIA" to "Wikipedia" and "WIKIMEDIA COMMONS" to "Wikimedia Commons"?

Capitals should not be used also on Android. That's not beautiful. Please change it for all platforms.

**Comment by dvdmrtnz** [comment]
Okay here it's how it will look now:

| Android | iOS |
|-|-|
|  |  |

File: iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift [hunk 2]
```diff
@@ -134,6 +138,18 @@ class PlacePageInfoViewController: UIViewController {
         self?.delegate?.didPressWebsite()
       }
     }
+    
+    if let wikipedia = placePageInfoData.wikipedia {
+      wikipediaView = createInfoItem("Wikipedia", icon: UIImage(named: "ic_placepage_wiki"), style: .link) { [weak self] in
```
Comment (biodranik) [inline iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift:143]: Wouldn't it be better to load these strings from data/strings/strings.txt and share between platforms?

**Review by vng — APPROVED** [review]

**Review by jimcarst — APPROVED** [review]

**Review by matheusgomesms — APPROVED** [review]

**Review by LaoshuBaby — APPROVED** [review]

**Pull request merged**